human.txt

```diff
@@ -37,6 +37,7 @@ michaelhwan
 mogodes
 monte9
 muan
+nosli
```

Comment:
Hey, I think you made a little mistake here :( Once you fix it we will be able to merge your PR